packages/create-app/template/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "{{ name }}",
+    "version": "0.0.1",
+    "description": "A GestaltJS web app",
+    "main": "index.js",
+    "scripts": {
+        "build": "echo 'building {{ name }}'"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "MIT",
+    "devDependencies": {},
+    "dependencies": {}
```

Comment:
We need to add `@gestaltjs/gestalt` as a dependency. You'll have to read the version of the CLI from the create-app package and pass it down to the Plop workflow to use. I believe you should be able to read it from the `package.json` by importing it directly in the code.